docs/toc.yml

```diff
@@ -1090,9 +1090,15 @@ items:
       - name: Overview
         href: reference/requirement-sets/powerpoint-api-requirement-sets.md
         displayName: PowerPoint
-      - name: PowerPoint preview APIs
+      - name: PowerPointApi Preview requirement set
         href: reference/requirement-sets/powerpoint-preview-apis.md
         displayName: PowerPoint
+      - name: PowerPointApi 1.1 requirement set
+        href: reference/requirement-sets/powerpoint-api-1-1-requirement-set.md
+        displayName: PowerPoint
+      - name: PowerPointApi 1.2 requirement set
```

Comment:
Excel and Word have the order of their requirement set pages flipped (e.g.: 1.2 is above 1.1). We should reconcile that difference one way or the other, but I'd recommend sticking with that convention.